readme.md

```diff
@@ -59,13 +59,15 @@ yarn upgrade
 
 ### Setup
 
-Write the configuration of the organization in a yml file in `org-configuration` directory.
-List of variables required in organization configuration:
-
-- name
-- slug
-- uuid: uuid of the organization
-- secret_key: token of the organization
+##### Add Organization configuration
+run `$ npm run add-org`
```

Comment:
run > Run
I'd add a semicolon at the end of the line. Check the result on github or in a preview.

Reply:
Run `$ npm run add-org`;
@nemesisdesign  it just adds the ; at the end of the line, is that what you want?